pvanalytics/quality/irradiance.py

```diff
@@ -390,3 +391,66 @@ def clearsky_limits(measured, clearsky, csi_max=1.1):
         max_clearsky_index=np.Inf
     )
     return util.check_limits(csi, upper_bound=csi_max, inclusive_upper=True)
+
+
+def _to_hours(freqstr):
+    if freqstr[0].isalpha():
+        freqstr = '1' + freqstr
+    return pd.to_timedelta(freqstr).seconds / 3600
+
+
+def _daily_total(series):
+    freq_hours = _to_hours(pd.infer_freq(series.index))
+    return series.resample('D').sum() * freq_hours
+
+
+def daily_limits(irrad, clearsky, daily_min=0.4, daily_max=1.25):
+    """Check that daily total irradiance is near the total clearsky irradiance.
+
+    Irradiance measurements on each day are integrated with the
+    left-hand rule and the total is compared to the total clearsky
+    irradiance for each day. If the ratio of total irradiance to total
+    clearsky is less than or equal to `daily_min` or greater than or
+    equal to `daily_max` then values on the day are marked False.
+
+    .. note::
```

Comment:
Should this go in the Notes section? Just asking

Reply:
Makes sense.